manuscript.md

```diff
@@ -423,19 +427,31 @@ generally be divided into two main categories (as suggested by
 @Berlow2004IntStr): 1) the strength of an interaction between individuals of
 each species, or 2) the effect that changes in one species population has on the
 dynamics of the other species. It can be measured as the effect over a period of
-time (in the units of biomass or energy flux [@Barnes2018EneFlu; @Brown2004MetThe]) or the
-relative importance of one species on another [@Heleno2014EcoNet;
+time (in the units of biomass or energy flux [@Barnes2018EneFlu; @Brown2004MetThe]) 
+or the relative importance of one species on another [@Heleno2014EcoNet;
 @Berlow2004IntStr; @Wootton2005MeaInt]. One recurring observation is that
 networks are often composed of many weak interactions and few strong
 interactions [@Berlow2004IntStr]. The distribution of interaction strength
 within a network effects its stability [@Neutel2002StaRea; @Ruiter1995EnePat]
 and functioning [@Duffy2002BioEco; @Montoya2003FooWeb], and serves to benefit
 multispecies models [@Wootton2005MeaInt].
 
+Quantifying interaction strenght in the field is (much like quantifying the 
```

Comment:
Estimating interaction strength is now here. We _could_ potentially still keep it as a section with a sub-heading but I think I prefer this